src/index.html

```diff
@@ -10,13 +10,53 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    >
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header>
+    <a
+      href="#"
+      class="logo"
+    >
+      <img
+        src="images/logo.png"
+        alt="logo"
+      />
+    </a>
+    <nav>
+      <ul>
+        <li>
+          <a
+            href="#"
+            class="is-active"
+          >
+            Apple
+          </a>
+        </li>
+        <li><a href="#">Samsung</a></li>
```

Comment:
Remember about correct indentation between parent and child elements. The 'a' tags within the 'li' elements should be indented.